lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs

```diff
@@ -634,24 +634,6 @@ spec = describe "SHELLEY_ADDRESSES" $ do
                 let walAddr = fst (addrs !! idx ^. #id) ^. (#getApiT . #unAddress)
                 walAddr `Expectations.shouldBe` genAddr
 
-    it "ANY_ADDRESS_POST_13 - at_least must make sense" $ \ctx -> do
```

Comment:
Why at_least can have no sense now? :sweat_smile: @paweljakubas

Reply:
ah, ok. so this test and more will be resurrected later as we will introduce required and recommended option in POST AnyAddress. Now we have JSON-parse validation and those are valid scripts. But they will not be of course when validation is available